pkg/console/config/config.go

```diff
@@ -177,23 +144,51 @@ func (w *watch) reloadTlsProfile() {
 		log.Log.V(1).Infof("Set ciphers: %s", strings.Join(cipherNames, ", "))
 	}
 
-	w.ciphers = ciphers
-	w.minTlsVersion = minVersion
+	c.ciphers = ciphers
+	c.minTlsVersion = minVersion
+}
+
+func (c *configWatch) reloadUseSaTokensLocked() {
+	c.useSaTokensError = nil
+
+	useSaTokensData, err := os.ReadFile(filepath.Join(c.configDir, c.useSaTokensName))
+	if errors.Is(err, os.ErrNotExist) {
+		c.useSaTokens = false
+		return
```

Comment:
Since `return` stops the execution, maybe add a log before that the file doesn't exist?

Reply:
done